newsfragments/1244.bugfix.rst

```diff
@@ -0,0 +1 @@
+Added helpful error message in the case the user passed incorrect type of function to "from_thread_run" & "from_thread_run_sync" functions.
```

Comment:
```suggestion
Added a helpful error message if a sync function is passed to `trio.from_thread.run_sync` or an async function to `trio.from_thread.run`.
```

Reworded for clarity and to link to the documentation of those functions (that's what the single backticks do).